target/linux/mediatek/image/filogic.mk

```diff
@@ -1090,7 +1090,9 @@ define Device/openwrt_one
   UBINIZE_OPTS := -E 5
   BLOCKSIZE := 128k
   PAGESIZE := 2048
-  UBINIZE_PARTS := fip=:$(STAGING_DIR_IMAGE)/mt7981_openwrt_one-snand-u-boot.fip recovery=:$(KDIR)/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb \
+  UBINIZE_PARTS := fip=:$(STAGING_DIR_IMAGE)/mt7981_openwrt_one-snand-u-boot.fip \
+		   $(if $(IB),recovery=:$(STAGING_DIR_IMAGE)/mediatek-filogic-openwrt_one-initramfs.itb,\
+			      recovery=:$(KDIR)/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb) \
```

Comment:
use `recovery=:$(KDIR)/tmp/$$(KERNEL_INITRAMFS_IMAGE)` instead please
cause `openwrt-mediatek-filogic-openwrt_one-initramfs.itb` could be rename to `xxxx-mediatek-filogic-openwrt_one-initramfs.itb` if `CONFIG_VERSION_DIST="xxxx"` set

Reply:
Correct, I will change accordingly. Thank you!